trace/trace-python-sample-opentelemetry/main.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import random
+import time
+
+from flask import Flask, redirect, url_for
+
+# [START trace_setup_python_configure]
+from opentelemetry import trace, propagators
+from opentelemetry.exporter.cloud_trace import CloudTraceSpanExporter
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import SimpleExportSpanProcessor
+
+
+def initialize_tracer(project_id):
+    trace.set_tracer_provider(TracerProvider())
+    cloud_trace_exporter = CloudTraceSpanExporter(project_id)
+    trace.get_tracer_provider().add_span_processor(
+        SimpleExportSpanProcessor(cloud_trace_exporter)
+    )
+    opentelemetry_tracer = trace.get_tracer(__name__)
+
+    return opentelemetry_tracer
+
+
+# [END trace_setup_python_configure]
+
+
+app = Flask(__name__)
+
+
+@app.route('/', methods=['GET'])
+def root():
+    return redirect(url_for('index'))
+
+
+# [START trace_setup_python_quickstart]
+@app.route('/index.html', methods=['GET'])
+def index():
+    tracer = app.config['TRACER']
+    with tracer.start_as_current_span("index"):
```

Comment:
in other strings, you're preferring single-quotes (as do I, thank you), be consistent